src/pages/reference/network/api.mdx

```diff
@@ -9,7 +9,7 @@ import { EndpointList } from "~/components/Docs";
 If you are a developer we highly recommend signing up for a free private
 endpoint through [AllThatNode](https://www.allthatnode.com/zetachain.dsrv),
 [BlockPI](https://blockpi.io/zeta), [InfStones](https://infstones.com/fast-api),
-[Blast API](https://blastapi.io/chains/zetachain) or one of our other RPC
+[Blast API](https://blastapi.io/chains/zetachain), [dRPC](https://drpc.org/chainlist/zeta-chain) or one of our other RPC
```

Comment:
Let's change it to `, [dRPC](https://drpc.org/chainlist/zeta-chain), or another RPC
provider.`